tests/components/matter/test_sensor.py

```diff
@@ -250,6 +260,24 @@ async def test_battery_sensor(
     assert entry
     assert entry.entity_category == EntityCategory.DIAGNOSTIC
 
+    """Test battery voltage sensor."""
+    entity_id = "sensor.eve_door_battery_voltage"
+    state = hass.states.get(entity_id)
+    assert state
+    assert state.state == "3558"
```

Comment:
The state should be in V and not in mV since we divide by 1000 before writing the state.